roles/figgy/meta/main.yml

```diff
@@ -24,4 +24,3 @@ dependencies:
   - role: "rails_app"
   - role: "ffmpeg"
   - role: "extra_path"
-  - role: "postgres"
```

Comment:
Do we need to add a dependency on the postgresql role (now that it's the only db-related role for postgres)?